package.json

```diff
@@ -73,11 +74,13 @@
     "singleQuote": true
   },
   "files": [
+    "lib-cjs",
     "lib",
     "styles",
     "images"
   ],
-  "main": "./lib/index.js",
+  "main": "./lib-cjs/index.js",
+  "module": "./lib/index.js",
```

Comment:
Modern module bundlers understand the convention that `main` in `package.json` specifies the CommonJS build of a package and the `module` field specifies the ES build. Browserify just ignores the new `module` field.